# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../metamail/90_mm-2.7-malloc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- mm2.7.orig/src/metamail/shared.c	2005-03-26 21:59:06.116190632 +0100
+++ mm2.7/src/metamail/shared.c	2005-03-26 22:08:01.887740976 +0100
@@ -11,7 +11,7 @@
 #include <sys/utsname.h>
 #endif
 
-extern char *malloc();
+extern void *malloc();
 char **Exceptions;
 int *NeedsPortableNewlines;
 int ExceptionsAlloced = 0, ExceptionsUsed = 0;
